allensdk/brain_observatory/ecephys/ecephys_project_api/http_engine.py

```diff
@@ -11,7 +11,7 @@
 import nest_asyncio
 from tqdm.auto import tqdm
 
-DEFAULT_TIMEOUT = 10 * 60  # seconds
+DEFAULT_TIMEOUT = 50 * 60  # set timeout to 50 minutes
```

Comment:
50 minutes seems excessive, have you encountered actually needing to wait 50 minutes for the warehouse to respond?

Maybe setting it to 20-30 minutes is more reasonable?

Reply:
I hit the timeout pretty frequently. I have 1GB internet download speed but the warehouse rarely delivers over 800kB/s. It's usually around 500kB/s so it takes about 3 minutes to download 100MB of data and about 30 minutes to download 1GB. But I think its probably a good idea to keep the default low. 20 minutes seems fine to me.